apps/studio/components/interfaces/Organization/BillingSettings/BillingSettings.tsx

```diff
@@ -1,4 +1,4 @@
-import { useNewLayout } from 'components/interfaces/App/FeaturePreview/FeaturePreviewContext'
```

Comment:
random..

Reply:
for visibility: small bug that slipped through in this [PR](https://github.com/supabase/supabase/pull/35060)
was renaming `useNewLayout` to `useIsNewLayoutEnabled` for consistency in there but (not sure how actually - figured the build would’ve failed) one file got missed here